seed/lib/superperms/orgs/decorators.py

```diff
@@ -17,6 +17,7 @@
     Organization,
     OrganizationUser
 )
+from seed.lib.superperms.orgs.permissions import get_org_id
```

Comment:
same method used by the OrgMixin, which is used for getting the org id inside of a view. This should be used to ensure each place is using the same strategy for getting the org id